.github/workflows/update.js

```diff
@@ -85,11 +95,17 @@ async function updatePackageJson(version) {
 async function commitAndPushTag(version) {
   const tag = `v${version}`;
   await git("add", "README.md", "package.json", "package-lock.json");
-  await git("commit", "-m", `"MAINT: upgrade to ${PACKAGE_NAME} ${version}"`);
+  await git("commit", "-m", `"${getMessage(version)}"`);
   await git("tag", tag);
   await git("push", "origin", tag);
 }
 
+async function mergePullRequest(versions) {
+  await git("push", "--set-upstream", "origin", getBranchName(versions.at(-1)));
+  await exec(`gh pr create --fill --title "${getMessage(versions.at(-1))}"`);
+  await exec("gh pr merge --rebase --delete-branch");
```

Comment:
Depending on the branch protection rules, whether PR approvals or status checks is required, this command may need to be modified accordingly.
```suggestion
  await exec("gh pr merge --auto --rebase --delete-branch");
```